pkg/apis/azure/types_cloudprofile.go

```diff
@@ -40,7 +40,7 @@ type DomainCount struct {
 	// Region is a region.
 	Region string
 	// Count is the count value for the respective domain count.
-	Count int
+	Count int32
```

Comment:
Why we need to convert the type to `int32`?

Reply:
It's a K8s API convention: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#primitive-types